packages/@react-spectrum/autocomplete/test/SearchAutocomplete.test.js

```diff
@@ -143,6 +143,7 @@ describe('SearchAutocomplete', function () {
     user = userEvent.setup({delay: null, pointerMap});
     jest.spyOn(window.HTMLElement.prototype, 'clientWidth', 'get').mockImplementation(() => 1000);
     jest.spyOn(window.HTMLElement.prototype, 'clientHeight', 'get').mockImplementation(() => 1000);
+    jest.spyOn(window.HTMLElement.prototype, 'scrollHeight', 'get').mockImplementation(() => 50);
```

Comment:
is this going to be problematic for users of v3 in their own tests?
will we need to call out in release?

Reply:
Probably something we'd like to call out but we also we have this section in are docs where we do mock the scrollHeight for virtualized components

Reply:
just my 2 cents, would be good to call out in the release notes. Just spent some time trying to figure out why one of my Picker tests were failing and it turned out it also needed this scrollHeight mock lol